crates/driver/src/print_build.rs

```diff
@@ -510,12 +513,23 @@ where
             } else {
                 let directory = to_directory(entry.to_string());
                 let full_file = opt.working_directory.join(&entry);
-                let name = to_file_name(&full_file).replace(".", "_");
+                let name = to_name_from_file_name(&to_file_name(&full_file), include_file_extension).unwrap();
```

Comment:
can we remove the unwrap or comment why it will never happen?

Reply:
It's highly unlikely but I added `unwrap_or()` and also commented in d5f87f4d9efa5e17aeb85d1e1e220ef650c86d4d